modules/sdk-coin-stt/test/unit/resources.ts

```diff
@@ -0,0 +1,29 @@
+import { KeyPair } from '@bitgo/abstract-eth';
+
+export const RECIPIENT = '0x19645032c7f1533395d44a629462e751084d3e4c';
+export const CONTRACT_ADDR = '0x8f977e912ef500548a0c3be6ddde9899f1199b81';
```

Comment:
I don't think CONTRACT_ADDR is required for tests. Please also look into other values in this file and if not needed, let's remove them